cli/stream_command.go

```diff
@@ -1705,6 +1705,7 @@ func (c *streamCmd) showStreamInfo(info *api.StreamInfo) {
 	}
 
 	fmt.Printf("     Active Consumers: %d\n", info.State.Consumers)
+	fmt.Printf("   Number of Subjects: %d\n", info.State.NumSubjects)
```

Comment:
I'd wrap this in an if >0 since older servers dont supply this information - and the display is getting giant already